docs/misc/changelog.rst

```diff
@@ -6,29 +6,28 @@ Changelog
 For download links, please look at `Github release page <https://github.com/hill-a/stable-baselines/releases>`_.
 
 
-Pre-Release 2.7.0a0 (WIP)
+Pre-Release 2.7.0 (WIP)
```

Comment:
This should be changed to Release 2.7.0 and drop the WIP before we release?

Reply:
Nice catch